FrontDesk/src/FrontDesk.Blazor/Shared/NavMenu.razor

```diff
@@ -12,21 +12,6 @@
                 <span class="oi oi-home" aria-hidden="true"></span> Schedule
             </NavLink>
         </li>
-        <li class="nav-item px-3">
-            <NavLink class="nav-link" href="clients">
-                <span class="oi oi-plus" aria-hidden="true"></span> Clients
-            </NavLink>
-        </li>
-        <li class="nav-item px-3">
-            <NavLink class="nav-link" href="doctors">
-                <span class="oi oi-plus" aria-hidden="true"></span> Doctors
-            </NavLink>
-        </li>
-        <li class="nav-item px-3">
-            <NavLink class="nav-link" href="rooms">
-                <span class="oi oi-plus" aria-hidden="true"></span> Rooms
-            </NavLink>
-        </li>
```

Comment:
I think the menu items should stay, but should link to the other app (ClinicManagement solution). Right?

Reply:
yes, it would be better.